app/models/medium.rb

```diff
@@ -0,0 +1,30 @@
+class Medium < ActiveRecord::Base
+  belongs_to :handle
+  validates :uri, :posted_at, presence: true
+
+  def self.create_tweet_medium(tweet_instance)
+    medium = Medium.new
+    medium.uri = tweet_instance.uri
+    medium.posted_at = tweet_instance.created_at
+    medium.save
+    medium.set_embed(tweet_instance)
+    return medium
+  end
```

Comment:
This looks like what should be an initialize method for the Medium class. Is there a reason you have implemented it with a class method instead?